job_templates/rediscover_host.erb

```diff
@@ -0,0 +1,50 @@
+<%#
+name: Rediscover node
+model: JobTemplate
+job_category: Discovery
+description_format: Writes grub2 configuration with "Rediscover host" entry.
+snippet: false
+template_inputs:
+- name: boot_interface
+  description: Boot interface MAC address (interactive TUI when blank)
+  required: false
+  input_type: user
+- name: set_default
+  description: Set as the default boot item (set to non-blank value)
+  required: false
+  input_type: user
```

Comment:
I wonder if we shouldn't have here options specified, to indicate that we only expect bool value here

```
  options: "yes"
```